store/postgres/src/dynds/mod.rs

```diff
@@ -78,16 +78,15 @@ pub(crate) fn update_offchain_status(
     }
 }
 
-/// The next causality region free to be assigned. Any higher number is also free to be assigned.
-pub(crate) fn causality_region_next_value(
+/// The maximum assigned causality region. Any higher number is therefore free to be assigned.
+pub(crate) fn causality_region_curr_val(
```

Comment:
could probably call this something like latest_causality_region

Reply:
This is just a nod to PG which uses this name for their equivalent sequence function https://www.postgresql.org/docs/current/functions-sequence.html